src/sensor/ping360.cpp

```diff
@@ -283,6 +350,60 @@ void Ping360::handleMessage(const ping_message& msg)
         break;
     }
 
+    case Ping360Id::AUTO_DEVICE_DATA: {
+        // Parse message
+        const ping360_auto_device_data autoDeviceData = *static_cast<const ping360_auto_device_data*>(&msg);
+
+        // Get angle to request next message
+        _angle = autoDeviceData.angle();
+
+        // Restart timer, if the channel allows it
+        if (link()->isWritable()) {
+            // Use 200ms for network delay
+            const int profileRunningTimeout = _angular_speed / _angularSpeedGradPerMs + 200;
+            _timeoutProfileMessage.start(profileRunningTimeout);
+        }
+
+        _data.resize(autoDeviceData.data_length());
+#pragma omp for
```

Comment:
move the pragma to the same line as the for

Reply:
while I do really - *really* - like openmp, perhaps we should note that it does not have a really good suppor for clang at the moment, and we should use another library instead, like tbb or qt concurrent.

Reply:
Also note that for clang11 (unreleased) the omp reached maturity

Reply:
@tcanabrava clang supports openmp for a long time already, I don't know exactly when but it's available for us since the beginning. I did a quick test with clang6 and it works.